Projects/Checkers/Game.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Checkers;
+
+public class Game
+{
+	private const int PiecesPerColor = 12;
+
+	public PieceColor Turn { get; private set; }
+	public Board Board { get; private set; }
+	public PieceColor? Winner { get; private set; }
+	public List<Player> Players { get; private set; }
+
+	public Game(int humanPlayerCount)
+	{
+		if (humanPlayerCount < 0 || 2 < humanPlayerCount) throw new ArgumentOutOfRangeException(nameof(humanPlayerCount));
+		Board = new Board();
+		Players = new()
+		{
+			new Player(humanPlayerCount >= 1, Black),
+			new Player(humanPlayerCount >= 2, White),
+		};
+		Turn = Black;
+		Winner = null;
+	}
+
+	public void PerformMove(Move move)
+	{
+		(move.PieceToMove.X, move.PieceToMove.Y) = move.To;
+		if ((move.PieceToMove.Color is Black && move.To.Y is 7) ||
+			(move.PieceToMove.Color is White && move.To.Y is 0))
+		{
+			move.PieceToMove.Promoted = true;
+		}
+		if (move.PieceToCapture is not null)
+		{
+			Board.Pieces.Remove(move.PieceToCapture);
+		}
+		if (move.PieceToCapture is not null &&
+			Board.GetPossibleMoves(move.PieceToMove).Any(move => move.PieceToCapture is not null))
```

Comment:
Might want to choose a different variable in the Any lambda rather than "move" since this is used in the method proper.

Reply:
fixed in 880198f8088f584ce70c31b053d68bdbe5bd576b